config/default/capm3/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: quay.io/metal3-io/cluster-api-provider-metal3:main
+      - image: quay.io/metal3-io/cluster-api-provider-metal3-amd64:v1beta1
```

Comment:
Remember to remote before merge!

Reply:
Oops, I have often seen that side-effect of building but did not recognize it after issuing code generation. Corrected.

Reply:
But maybe this was not the purpose of your comment.

Reply:
Yeah it is unfortunate that the current way of deploying and testing relies on making changes to these manifests :grimacing: 
I just wanted to make sure we remember to remove any changes to this line. Looks like you now accidentally removed the `:main` at the end.

Reply:
Corrected now, sorry for the noice (again).